client/src/Component/signin/index.jsx

```diff
@@ -0,0 +1,163 @@
+import { React, useState } from 'react';
+import {
+  Layout,
+  Row,
+  Col,
+  Image,
+  Form,
+  Button,
+  Input,
+  message,
+  Typography,
+  Anchor,
+} from 'antd';
+
+import { useDispatch } from 'react-redux';
+
+import { GoogleOutlined } from '@ant-design/icons';
+
+import axios from 'axios';
+import 'antd/dist/antd.min.css';
+import './index.css';
+import { sign } from '../../redux/feature/user/userSlice';
+
+const { Title, Text } = Typography;
+const { Item } = Form;
+const { Link } = Anchor;
+
+const {
+  Content,
+} = Layout;
+
+function Signin() {
+  const [userData, setUserData] = useState({ email: '', password: '' });
+  const dispatch = useDispatch();
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setUserData({
+      ...userData,
+      [name]: value,
+    });
+  };
+
+  const passwordValidation = () => ({
+    validator(_, value) {
+      if (value.length >= 6) {
+        return Promise.resolve();
+      }
+      return Promise.reject(new Error('password should be at least 6 characters'));
+    },
+  });
+
+  const login = async () => {
+    try {
+      const { data: { data } } = await axios.post('/api/login', userData);
+      dispatch(sign(data));
+      message.success(`Welcome back ${data.name}`);
+    } catch ({ response: { data: { message: errorMessage } } }) {
+      message.error({
+        content: errorMessage,
+      });
+    }
+  };
+
+  return (
+    <Layout>
+      {/* --- Navbar --- */}
+      <Content>
+        <Row>
+          <Col span={12} className="login_image">
+            <Image
+              style={{ height: '100vh', width: '100vh' }}
```

Comment:
Move the style, and maybe you need to control the the size from the container instead of given the card fixed width or height